### PR TITLE
Feature/new loading

### LINKED ISSUE #23: Add a Loader for all the pages

### PR DESCRIPTION
**Pull request by aadarsh10c**
Removed the page loader , kept the login/signup loader .
Closes #23

### CODE REVIEW

File: client/src/components/LoginButton.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import React from 'react'
```
Comment (shaurya35) [inline client/src/components/LoginButton.jsx:1]: change this component to loader and move this in a separate folder
/components/loaders/Loader.jsx something like that for structurize format

**Comment by shaurya35** [comment]
@aadarsh10c you are good to go, but since this is a production level website, there are some changes that needs to be fixed,
also please add a function that if login is clicked without entering the data, it wont show loader
image

File: client/src/components/Auth/Login.jsx [hunk 2]
```diff
@@ -13,6 +14,13 @@ const Login = () => {
     await login(email, password);
   };
 
+  useEffect(() => {
```
[the remark this answers is not in the document]
Comment (aadarsh10c) [reply client/src/components/Auth/Login.jsx:17]: done

File: client/src/components/Auth/Login.jsx [hunk 3]
```diff
@@ -30,9 +38,13 @@ const Login = () => {
             onChange={(e) => setPassword(e.target.value)}
             className="outfit"
           />
-          <button type="submit" disabled={isLoading} className="outfit">
-            Login
-          </button>
+          <LoginButton
```
[the remark this answers is not in the document]
Comment (aadarsh10c) [reply client/src/components/Auth/Login.jsx:41]: done

File: client/src/components/LoginButton.jsx (hunk 1 above)
Comment (aadarsh10c) [reply client/src/components/LoginButton.jsx:1]: done

**Comment by aadarsh10c** [comment]
@shaurya35 , I have added the changes

**Comment by shaurya35** [comment]
@aadarsh10c great work

**Comment by shaurya35** [comment]
@aadarsh10c thank you for contributing in this repo

**Comment by shaurya35** [comment]
image
are you getting this error as well?

**Comment by aadarsh10c** [comment]
> image are you getting this error as well?

no i did not

**Comment by shaurya35** [comment]
@aadarsh10c fixed good to go

**Pull request merged**